admin/company/settings/scim.md

```diff
@@ -16,8 +16,8 @@ You must make sure you have [configured SSO](sso.md) before you enable SCIM. Enf
 
 1. Sign in to [Docker Admin](https://admin.docker.com){: target="_blank" rel="noopener" class="_"}.
 2. In the left navigation, select your company in the drop-down menu.
-3. Select **Settings**.
-4. In the **Single Sign-On Connection** table, select the **Actions** icon and **Setup SCIM**.
+3. Select **SSO & SCIM**.
+4. In the table, select the **Actions** icon and **Setup SCIM**.
```

Comment:
Is the UI copy in prod different than stage? I can only test this on in staging and there I see **Actions** icon > **Enable SCIM**.